services/certificate-api/src/routes/certificate_controller.js

```diff
@@ -5,6 +5,8 @@ const puppeteer = require('puppeteer');
 const QRCode = require('qrcode');
 const JSZip = require("jszip");
 const { default: axios } = require('axios');
+const QRTYPE = 'URL-W3C-VC';
```

Comment:
rename the const name to URL_BASED_QR or something better?

Reply:
ok